appinventor/appengine/src/com/google/appinventor/client/widgets/properties/ColorChoicePropertyEditor.java

```diff
@@ -310,9 +203,11 @@ protected void updateValue() {
    * @param b Blue channel value of the color in [0, 255]
    * @return HTML string for the Custom... dropdown entry
    */
-  private static String makeCustomHTML(double a, int r, int g, int b) {
-    return "<span style=\"background:rgba(" + r + "," + g + "," + b + "," + a + "); border:1px solid black; " +
-        "width:1em; height:1em\">&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;&nbsp;" + MESSAGES.customEllipsis();
+  private static String makeCustomHTML(long argbValue, double a, int r, int g, int b) {
+    String hex = argbToHex(argbValue);
+    String displayValue = hex.endsWith("FF") ? hex.substring(0, hex.length() - 2) : hex;
+    return "<span style=\"background:rgba(" + r + "," + g + "," + b + "," + a + "); display: inline-block; " +
+            "width:15px; height:15px; border-radius:200px; border: 1px solid var(--border-color);\">&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;&nbsp;" + displayValue;
```

Comment:
So I have some reservations here...

1. In the old picker, we use the values of #00000000 and #00ffffff to indicate special cases of Default and None. By only ever showing the hex value, this information gets lost.
2. In a similar vein, regular colors (red, etc.) show their i18n names in the old version. If a novice who is color blind, the written name is probably more useful than the hex value.